core/controllers/question.py

```diff
@@ -88,34 +62,18 @@ def post(self):
         """Handles POST requests."""
         if not self.payload.get('question'):
             raise self.PageNotFoundException
-        if not self.payload.get('skill_id'):
-            raise self.PageNotFoundException
         question = question_domain.Question.from_dict(
             self.payload.get('question'))
-        skill_id = self.payload.get('skill_id')
         question_id = question_services.add_question(
             self.user_id, question)
-        question_services.add_question_id_to_skill(
-            question_id, question.collection_id, skill_id, self.user_id)
         return self.render_json({
             'question_id': question_id
         })
 
 
 class QuestionManagerHandler(base.BaseHandler):
-    """This handler completes requests for question summaries."""
-
-    @acl_decorators.can_access_moderator_page
-    def get(self):
-        """Handles GET requests."""
-        collection_id = self.request.get('collection_id')
-        if not collection_id:
-            raise self.PageNotFoundException
-        question_summaries = (
-            question_services.get_question_summaries_for_collection(
-                collection_id))
-        return self.render_json({
-            'question_summary_dicts': [
-                question_summary.to_dict()
-                for question_summary in question_summaries]
-        })
+    """This handler completes requests for question summaries.
```

Comment:
You can just delete this handler altogether if it's not used (make sure to check its callers too).

Reply:
Done